db/migrate/20160116230959_add_details_to_movies.rb

```diff
@@ -0,0 +1,6 @@
+class AddDetailsToMovies < ActiveRecord::Migration
+  def change
+    add_column :movies, :image, :string
+    add_column :movies, :thumbnail, :string
```

Comment:
NOTE:
string is actually varchar(255) in DB and 255 characters can be eaten by AWS s3 url in no time. This will not happen in your case but in general when you have link stored in DB use text.